package.json

```diff
@@ -99,13 +106,12 @@
   ],
   "scripts": {
     "precommit": "lint-staged",
+    "clean": "rm -rf dist",
```

Comment:
This may or may not work on Windows. Can't remember if we need [**rimraf**](http://npm.im/rimraf) for cross-OS compatibility.